### PR TITLE
New chyron color fields in projector

### PR DESCRIPTION
**Pull request by luisa-beerboom**
See OpenSlides/openslides-backend#2437

### CODE REVIEW

File: models.yml [hunk 1]
```diff
@@ -3874,10 +3874,18 @@ projector:
     type: color
     default: "#317796"
     restriction_mode: A
+  chyron_background_color_2:
+    type: color
+    default: "#134768"
```
Comment (reiterl) [inline models.yml:3879]: Where does this default come from?

**Review by reiterl — COMMENTED** [review]
I am not sure, if the default is correct.

File: models.yml (hunk 1 above)
Comment (luisa-beerboom) [reply models.yml:3879]: Same colour as the global headbar in default theme

Comment (reiterl) [reply models.yml:3879]: Okay, I think it should be fine then. Nice.

**Review by reiterl — APPROVED** [review]
LGTM

**Pull request merged**